packages/core/src/VNodes/VNode.ts

```diff
@@ -57,12 +57,18 @@ export class VNode {
     get atomic(): boolean {
         return (this.constructor as typeof VNode).atomic;
     }
+    get attributesObject(): { [key: string]: AttributeValue } {
```

Comment:
This is a crappy idea, forget it. It was just for the DevTools to display the attributes properly because there still is that Owl bug that prevents us from using `get` on the map but there has to be a better way anyway.